logic.py

```diff
@@ -21,14 +23,32 @@ def tuple2list(ExTuple):
     return ExList 
 
 def SQLQuery(statment):
-    cursor.execute(statment)
-    conn.commit()
-    data = logic.tuple2list(cursor.fetchall()) 
-    return data
+    try:
+        cursor.execute(statment)
+        conn.commit()
+        data = logic.tuple2list(cursor.fetchall())
+        status = "OKAY"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+        return data
+    except:
+        status = "ERROR"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+    
 
 def SQLInsertQuery(statment):
-    cursor.execute(statment)
-    conn.commit()
+    try:
+        cursor.execute(statment)
+        conn.commit()
+        status = "OKAY"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
+        return data
+    except:
+        status = "ERROR"
+        purpose = "PRODUCTION"
+        SQL_log(statment,status,purpose)
```

Comment:
Can we make a separate function for Try and exception and then use it with these functions so that we can avoid code repetition?

Reply:
I am not really sure how to do this, do we have any examples of how to turn a try and expect block to a function?

Reply:
I think I have seen one in my previous course.  If the code is the same for both functions, we can try to make a separate function of this code and call it in SQL function. But if it's not possible then, this should be fine.